source/code/troubleshooter/modules/main.py

```diff
@@ -7,7 +7,7 @@
 from connect.connect           import check_connection
 from heartbeat.heartbeat       import check_heartbeat
 from high_cpu_mem.high_cpu_mem import check_high_cpu_memory
-from syslog1.syslog            import check_syslog
+from syslog_tst.syslog            import check_syslog
```

Comment:
nit indent

Reply:
Sorry forgot to fix the indentation I had set up after changing `syslog1` to `syslog_tst`, fixed it here:
https://github.com/microsoft/OMS-Agent-for-Linux/blob/1fcd4dd0d1cef4e18b71d295a048a3e4b7412dc9/source/code/troubleshooter/modules/main.py#L10
and here:
https://github.com/microsoft/OMS-Agent-for-Linux/blob/1fcd4dd0d1cef4e18b71d295a048a3e4b7412dc9/installer/datafiles/base_omsagent.data#L176-L179